docs/concepts/persistence.md

```diff
@@ -83,3 +82,19 @@ Some available pinning service providers are:
 - [Crust Network](https://crust.network/)
 
 See how to [work with remote pinning services](../how-to/work-with-pinning-services.md).
+
+## Long-term storage
+
+Storing data using a personal IPFS node is easy, but it can be inconvenient since you have to manage your own hardware. This problem gave rise to _pinning services_, paid services that allow you to upload your data to a remotely hosted IPFS node and retrieve it whenever you want. However, while paying a pinning service to store data is a convenient workaround, it still requires someone to bear the cost of storing that data. If that one sponsor stops paying for that pinning, the content may be lost entirely. While IPFS guarantees that any content on the network is discoverable, it doesn't guarantee that any content is persistently available. This is where [Filecoin](https://filecoin.io) comes in.
+
+### Storing data with Filecoin
+
+[Filecoin](https://filecoin.io) is a decentralized storage network in which storage providers rent their storage space to clients. The client and the storage provider agree on how much data will be stored, for how long, and at what cost. This agreement is called a _deal_. Once both parties agree to a deal, the client sends the data to the storage provider, who periodically verifies that they are correctly storing the data. When the client wants the data back, they send a request to the storage provider, who initiates the data transfer back to the client. For more information on how Filecoin works, head over to the [official Filecoin documentation →](https://docs.filecoin.io/about-filecoin/how-filecoin-works/)
+
+Filecoin provides users with a dependable, long-term storage solution. However, there are some limitations to consider. The retrieval process is not always as fast as an IPFS pinning service, and the minimum file size accepted by a Filecoin storage provider can be several GiB. Also, the process for creating a storage deal may seem complicated to new users who aren't familiar with blockchain transactions or simply aren't comfortable working within a command line. If these are potential blockers for you, you may want to use [Web3.Storage](https://web3.storage).
```

Comment:
Instead of the last line I'd say: 

"Fortunately, there is a growing community of tools and service providers that help simplify the process of making content available over IPFS while persisting the data via FIlecoin. Below are a few options:

- Web3.Storage
- Estuary
- Powergate
- Chainsafe Storage
- Fleek

Reply:
@jessicaschilling

Reply:
Resolved in https://github.com/ipfs/ipfs-docs/pull/847/commits/539a1ceb2fe0ae879acb6857005f6fa3686c6e8c.